api/.env.example

```diff
@@ -0,0 +1 @@
+ADMIN_JWT_SECRET=8Gkwzv3B8BOp5V/oVC1TN9N0Xs/27lZ5+scRM4T+6Fr+JuZshJduPzoEmvgHLQMJqrDlNBYYnyrdAHtcE48vBA==
```

Comment:
I would prefer the value to be empty so the user doesn't copy without changing the value. Can you also add a section in the README to say to copy the file and set a value ?

Reply:
The main readme says that you need to run `cd api && yarn && yarn run seed && yarn develop`
And the `seed` function generates this file - so in fact, I just removed the file .env example file because it's not really needed in fact.